src/style.css

```
@@ -304,3 +303,89 @@
 .rdp-focusable {
   cursor: pointer;
 }
+
+@keyframes rdp-animation_slide_in_left_keyframes {
+  0% { transform: translateX(-100%); }
+  100% { transform: translateX(0); }
+}
+
+@keyframes rdp-animation_slide_in_right_keyframes {
+  0% { transform: translateX(100%); }
+  100% { transform: translateX(0); }
+}
+
+@keyframes rdp-animation_slide_out_left_keyframes {
+  0% { transform: translateX(0); }
+  100% { transform: translateX(-100%); }
+}
+
+@keyframes rdp-animation_slide_out_right_keyframes {
+  0% { transform: translateX(0); }
+  100% { transform: translateX(100%); }
+}
+
+.rdp-animation_new_month_is_after {
+  animation: rdp-animation_slide_in_right_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+.rdp-animation_old_month_is_before {
+  animation: rdp-animation_slide_out_left_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+.rdp-animation_new_month_is_before {
+  animation: rdp-animation_slide_in_left_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+.rdp-animation_old_month_is_after {
+  animation: rdp-animation_slide_out_right_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+[dir="rtl"] .rdp-animation_new_month_is_after {
+  animation: rdp-animation_slide_in_left_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+[dir="rtl"] .rdp-animation_old_month_is_before {
+  animation: rdp-animation_slide_out_right_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+[dir="rtl"] .rdp-animation_new_month_is_before {
+  animation: rdp-animation_slide_in_right_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+[dir="rtl"] .rdp-animation_old_month_is_after {
+  animation: rdp-animation_slide_out_left_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+@keyframes rdp-animation_fade_in_keyframes {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes rdp-animation_fade_out_keyframes {
+  from {
+    opacity: 1;
+  }
+  to {
+    opacity: 0;
+  }
+}
+
+.rdp-animation_new_month_caption_is_after {
+  animation: rdp-animation_fade_in_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+.rdp-animation_old_month_caption_is_before {
+  animation: rdp-animation_fade_out_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+.rdp-animation_new_month_caption_is_before {
+  animation: rdp-animation_fade_in_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
+
+.rdp-animation_old_month_caption_is_after {
+  animation: rdp-animation_fade_out_keyframes var(--rdp-animation_duration) var(--rdp-animation_timing) forwards;
+}
```

Missing EOF new line: 

```suggestion
}

```